content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Installing Istio for virtual machine connectivity
+description: Install an Istio mesh such that a virtual machine can connect to it and configure the virtual machine.
+weight: 3
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to install an Istio control plane
+[multicluster deployment](/docs/ops/deployment/deployment-models/#multiple-clusters) and connect your first virtual machine.
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Installing the Istio control plane
+
+1. Set the name of your cluster as an environment variable. Please be aware
+    that cluster names may be autogenerated by your Kubernetes cloud provider.
+    Therefore, the cluster name you specify during creation of your cluster may
+    be different than the name of the cluster expected here. This will be evident
+    by invalid x509 certificate errors in the Istio logs after completing
+    installation.
```

Comment:
Will this really break anything? As far as I can tell from reading the docs this is just where files are stored and it could be literally anything?

Reply:
I don't think it ever broke anything - and don't know what the this is about...

Reply:
The cluster name had to match in my testing. If it didn't, x509 cert errors were displayed. I would encourage you to validate my results.

Reply:
@sdake how can this happen? Its just used as the folder name, right? We could call this `vm-stuff` if we wanted and it would be the same. I think its likely you ran into a different issue

Reply:
@howardjohn please reference: https://github.com/istio/istio/blob/master/samples/certs/Makefile#L136-L184

Cert generation uses the name passed to make (try `make help` in this directory) to generate certificates. I will verify again, although I have tried it twice already with an x509 cert error.

Reply:
As best I can tell the its just specifying the folder? The only thing its used in is the L field which I don't think would cause these. The only error that could possibly be related to cluster name (to the best of my knowledge) is for multicluster stuff since the MC secret has the cluster name - otherwise I don't see how it could matter as long as we use the same value throughout

Reply:
Will test more thoroughly and get back to you.

Reply:
Will test more thoroughly and get back to you.

Reply:
agree, it is just a folder name

Reply:
addressed.